clang/lib/Driver/ToolChains/WebAssembly.cpp

```diff
@@ -968,6 +968,9 @@ void cheerp::CheerpCompiler::ConstructJob(Compilation &C, const JobAction &JA,
     cheerpGlobalPrefix->render(Args, CmdArgs);
   if(Arg *cheerpHeapSize = Args.getLastArg(options::OPT_cheerp_linear_heap_size))
     cheerpHeapSize->render(Args, CmdArgs);
+  else if (Args.hasArg(options::OPT_fsanitize_EQ) &&
+           Args.getLastArg(options::OPT_fsanitize_EQ)->containsValue("address"))
+    CmdArgs.push_back("-cheerp-linear-heap-size=2000"); // ASan requires quite a bit of memory to be effective
```

Comment:
shouldn't we check if the option was also manually specified?

Reply:
It gets checked right before in the if case? Or do you mean giving a warning if you specify a low amount of memory?